umap/views.py

```diff
@@ -634,6 +668,20 @@ def post(self, *args, **kwargs):
         return response
 
 
+class MapStar(View):
+
+    def post(self, *args, **kwargs):
+        map_inst = get_object_or_404(Map, pk=kwargs['map_id'])
+        qs = Star.objects.filter(map=map_inst, by=self.request.user)
+        if qs.exists():
+            qs.delete()
+            status = False
+        else:
+            Star(map=map_inst, by=self.request.user).save()
```

Comment:
```suggestion
            Star.objects.create(map=map_inst, by=self.request.user)
```

I think this is a more djangonic(?) way (optional).